### DISCUSSION TITLE
Loading Up PDF and Requesting it to be read

### DISCUSSION THREAD

**Discussion by KenDiop** [chamilo/chamilo-lms#5861] [General]
I am a new user.  I have uploaded a pdf file to the created course.  When I am in the Student View I can see the pdf.  When the student clicks on it the pdf file, it opens.  I then highlight the text and right mouse click "Read aloud section".  I can then see in the top left a volume type symbol.  It looks like it is playing something, but I can't hear anything.  I can see the start and stop button but I can't use it.

If I open the pdf in my computer, it can read it aloud.  

Can you help?

Thanks

**Comment by LudiscapeXApi** [comment]
Make sure that your browser tab allows sound playback. Also, the PDF reader on your PC may not be the same as the one in your browser. Another solution would be to convert your PDF into an HTML (cloudconvert ?) file and integrate it into your course, so that the accessibility tools work properly.
